metadata_service/proxy/neo4j_proxy.py

```diff
@@ -497,20 +510,49 @@ def delete_owner(self, *,
             tx.commit()
             tx.close()
 
+    @timer_with_counter
+    def get_tag(self, *, table_uri: str, tag_type: str = 'default') -> List:
```

Comment:
Is there anywhere calling this method? Was not able to find it..

Reply:
will remove it.